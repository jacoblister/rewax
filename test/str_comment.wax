(func str_comment_test
  (print "str_comment_test:")
  (let comment str "; line comment")
  (print comment)
)

(func main (result int)
  (call str_comment_test)
  (return 0)
)