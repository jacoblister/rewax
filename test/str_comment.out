str_comment_test:
; line comment
